src/Microsoft.Azure.SignalR/HubHost/ServerLifetimeManager.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.SignalR.HubHost

```

Comment:
>HubHost [](start = 34, length = 7)

use `Microsoft.Azure.SignalR` namespace and remove this one in `DependencyInjection`